darwin/fixed.m

```diff
@@ -0,0 +1,201 @@
+// 15 august 2015
+#import "uipriv_darwin.h"
+
+// TODO hiding all stretchy controls still hugs trailing edge
```

Comment:
Does this TODO have anything to do with this file?

Reply:
No. I copied box.m and then modified that since I was unfamiliar with the layout and I forgot to take that out.